client/src/components/StatsDummy/Dummy.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const StatsDummy: React.FC = () => {
+  const { t }: any = useTranslation();
```

Comment:
what's "t"?

Reply:
The function which is responsible for localization -i18next library-